setup.php

```diff
@@ -35,7 +35,7 @@
  ----------------------------------------------------------------------
  */
 
-define ('PLUGIN_GENERICOBJECT_VERSION', '2.9.2');
+define ('PLUGIN_GENERICOBJECT_VERSION', '2.9.3');
```

Comment:
Please not change version inside a PR. This is done during the release process.
```suggestion
define ('PLUGIN_GENERICOBJECT_VERSION', '2.9.2');
```